backend/src/nodes/nodes/image_utility/convolve.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+import numpy as np
+
+from . import category as ImageUtilityCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput, TextInput
+from ...properties.outputs import ImageOutput
+from ...properties import expression
+
+
+@NodeFactory.register("chainner:image:image_convolve")
+class ImageConvolveNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Convolves input image with input kernel (kernel values separated by commas)"
+        self.inputs = [
+            ImageInput(),
+            TextInput("Kernel String"),
+            NumberInput("Kernel Width/Height", minimum=0, default=3),
+            NumberInput("Padding", minimum=0, default=0),
```

Comment:
Is this parameter really necessary? I mean, we already have a node to do this (Create Border).

Reply:
Padding is something generally used at the same step as convolution. While normally I'd say let the other node do it, since this is standard for convolution i say just keep it.

Reply:
For reference for this and stride, check this out https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html

Reply:
Oh, in that case, I'm fine with both padding and stride.